.github/workflows/test_pkg_gen.yaml

```diff
@@ -20,8 +20,10 @@ jobs:
       fail-fast: false
       matrix:
         config:
-          - {os: windows-latest, r: 'release', rust-version: 'stable-gnu', rtools-version: '42'}
-          - {os: windows-latest, r: 'devel',   rust-version: 'stable-gnu', rtools-version: '42'}
+          - {os: windows-latest, r: 'release', rust-version: 'stable-gnu',  rtools-version: '42'}
+          - {os: windows-latest, r: 'release', rust-version: 'stable-msvc', rtools-version: '42'}
+          - {os: windows-latest, r: 'devel',   rust-version: 'stable-gnu',  rtools-version: '42'}
+          - {os: windows-latest, r: 'devel',   rust-version: 'stable-msvc', rtools-version: '42'}
```

Comment:
Since you changed toolchain to `stable` in makevars, I think even if you specify `rust-version: 'stable-gnu'`, it still uses `msvc`. And I think it works because `msvc` is available by default.

Reply:
No. The default is set here.

https://github.com/extendr/rextendr/blob/f627e943bfb507192b5599d1e331448c80049906/.github/workflows/R-CMD-check.yaml#L42-L43

Reply:
Wnat I mean is that in `Makevars` you change toolchain to `msvc`. https://github.com/extendr/rextendr/blob/f627e943bfb507192b5599d1e331448c80049906/inst/templates/Makevars.ucrt#L2

So even if you install `gnu` toolchain, when executing `Makevars`, we are using `msvc`. Unless the env var `TOOLCHAIN` is overriden before.

Reply:
Ah, sorry. I thought `stable` can mean GNU or MSVC depending on the default toolchain. I'll fix.